run-checks

```diff
@@ -231,12 +231,9 @@ if [ "$STATIC" = 1 ]; then
     if ! command -v misspell >/dev/null; then
         go get -u github.com/client9/misspell/cmd/misspell
     fi
-    for file in *; do
-        if [ "$file" = "vendor" ] || [ "$file" = "po" ]; then
-            continue
-        fi
-        misspell -error -i auther,PROCES,PROCESSS,proces,processs,exportfs "$file"
-    done
+    MISSPELL_IGNORE="auther,PROCES,PROCESSS,proces,processs,exportfs"
```

Comment:
Hmm, why are we ignoring those specific words then?

Reply:
The PROCES stuff is from the seccomp tests (PRIO_PROCESS)
The exportfs is something from our nfs test that sounds like "exports"
The auter is actually a mispell in our changelog that we should probably fix, so nice catch!

Reply:
It would be good, instead, to have a per-line exception comments or something like that. Otherwise we can spell process incorrectly anywhere in the tree without consequence.

Reply:
Yeah, I don't think misspell supports it fine-grained like this unfortunately.

Reply:
I had a look and it doesn't. Boo